testing/regress/hpcc/regression/regress.py

```diff
@@ -343,20 +343,21 @@ def StartTimeoutThread(self):
         self.timeoutThread.start()
 
     def CheckTimeout(self, cnt,  threadId,  query):
+        defSleepTime=0.1
```

Comment:
Was this change (1.0 to 0.1) intentional ? desirable? It's not really part of the Jira ticket this PR is claiming to address.